main.qml

```diff
@@ -1953,9 +1953,13 @@ ApplicationWindow {
         if (parts.length == 4) {
           var version = parts[0]
           var hash = parts[1]
-          var user_url = parts[2]
-          var auto_url = parts[3]
-          var msg = qsTr("New version of monero-wallet-gui is available: %1<br>%2").arg(version).arg(user_url) + translationManager.emptyString
+          //var user_url = parts[2]
+          //var auto_url = parts[3]
+          var osBuildTag = isMac ? "mac-x64" : isWindows ? "win-x64" : isLinux ? "linux-x64" : "unknownBuildTag"
```

Comment:
Return or throw a error in the unknown/unrecognized OS case.

Reply:
Excellent idea. Thanks.